platformio/template_renderer.py

```diff
@@ -0,0 +1,24 @@
+import jinja2
```

Comment:
Would you mind adding a top level file comment here explaining what this does and the context of its use for the posterity?

Reply:
Done